frontend/src/store/auth/slice.ts

```diff
@@ -7,7 +7,7 @@ type AuthState = {
 
 const initialState: AuthState = {
   user: null,
-  token: null,
+  token: localStorage.getItem('access-token') || null,
```

Comment:
Please use `StorageKey` added in - https://github.com/BinaryStudioAcademy/bsa-2022-autoline/pull/76